doc/_static/mpl.css

```diff
@@ -1,3 +1,62 @@
+:root {
+  /* START COLOR PALETTE */
+
+  /* viridis solid colors */
+  --dark-purple: #440154;
+  --purple: #482475;
```

Comment:
We define these somewhere, I'm pretty sure.  Maybe they were moved to mpl-sphinx-theme?

Reply:
You're right, they're defined in pydata-sphinx-theme; I'm not sure we need all of these.

Reply:
Of the variables actually used below, there is neither `--border-radius` nor `--shadow`. There is obviously no `--viridis-purple`. There is `--gray:#6c757d;--gray-dark:#343a40;` but not `--light-gray`, and `--blue` is instead `#007bff`.